src/functions/types.ts

```diff
@@ -82,7 +82,8 @@ type FunctionInputRuntimeType<
       | typeof SlackSchemaTypes.user_id
       | typeof SlackSchemaTypes.usergroup_id
       | typeof SlackSchemaTypes.channel_id
-      | typeof SlackSchemaTypes.date ? string
+      | typeof SlackSchemaTypes.date
+      | typeof SlackSchemaTypes.rich_text ? string
```

Comment:
Could we set `rich_text` to `UnknownRuntimeType` instead of `string`? This will treat it as `any`

Reply:
Should we keep date as `typeof SlackSchemaTypes.date ? string`? I didn't realize how the ? operator is working here